atest/testdata/parsing/suite_names/__init__.robot

```diff
@@ -0,0 +1,4 @@
+*** Settings ***
+Force Tags      suite_naming
+
+Suite Name      Parent Init Suite
```

Comment:
This setting should be just `Name`, not `Suite Name`.

No need to use tags here. If there was a need, should use `Test Tags` instead of deprecated `Force Tags`. Finally, we don't generally use empty lines between settings and following the same style would keep tests consistent.